README.md

```diff
@@ -14,5 +11,32 @@ The scripts install the following:
 There are also scripts for installing software individually. Software that currently has individual install scripts:
 - [TEMOA](https://github.com/TemoaProject/temoa)
 
+# Requirements
+ - The user must fork all of the listed software tools (unless otherwise specified) listed below for the scripts to run successfully. 
+ - The user must have [conda]() installed. Currently, the software only works if with anaconda. The `anaconda3` must be located in `home`. I recommend using Sam Dotson's script [ubuntu-post-installer](https://github.com/samgdotson/ubuntu-post-installer) to do this, as well as set up a few other useful pieces of software on ubuntu)
+
+# Installation
+Assuming you have set up [ssh keys](https://docs.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh), run the following in your terminal
+``
+git clone git@github.com:yardasol/ubuntu-nuclear-software-installer
+``
+Boom. You're done
+
+# Usage
+***NOTE:*** The scripts have my directory structure hard-coded in right now, so the instructions below will not work as-is without some tinkering on your part. This will be changed in the near future to be more generalized.
+
+To use the software, run the following in your terminal
+``
+cd ubuntu-nuclear-software-installer
+source ubuntu-nuclear-software-installer.sh
+``
+Boom. You're done.
+
+# Contributing
+Please read [CONTRIBUTING.md](https://github.com/yardasol/ubutnu-nuclear-software-installer/blob/main/CONTRIBUTING.md) before making any changes
```

Comment:
The link to the contributing document won't work as-is, but should once I merge to main